tests/Tests/400__ods_dashboard/415__ods_custom_docs.robot

```diff
@@ -0,0 +1,167 @@
+*** Settings ***
+Library          OpenShiftLibrary
+Resource         ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Resource         ../../Resources/Page/LoginPage.robot
+Resource         ../../Resources/Page/OCPLogin/OCPLogin.robot
+Resource         ../../Resources/Page/ODH/ODHDashboard/ResourcesPage.resource
+Resource          ../../Resources/Page/ODH/ODHDashboard/ODHDashboardResources.resource
+Suite Setup      Custom Doc Suite Setup
+Suite Teardown   Custom Doc Suite Taerdown
+
+
+*** Variables ***
+${QS_YAML}=                  tests/Resources/Files/custom_quickstart.yaml
+${APP_YAML}=                 tests/Resources/Files/custom_app.yaml
+${HOWTO_YAML}=               tests/Resources/Files/custom_doc_howto.yaml
+${TUTORIAL_YAML}=            tests/Resources/Files/custom_doc_tutorial.yaml
+&{EXPECTED_ITEMS_TITLES}=    quickstart=TEST - Custom Quick Start
+...                          application=TEST - Custom ODS-CI Application
+...                          howto=TEST - Custom How-To Documentation
+...                          tutorial=TEST - Custom Tutorial Documentation
+${CUSTOM_APP_DICT_PATH}=     tests/Resources/Files/CustomAppInfoDictionary.json
+
+
+*** Test Cases ***
+Verify Documentation Items Can Be Added Using Odh CRDs
+    [Documentation]     Verified it is possible to create QuickStarts,Tutorials,How-to and Application
+    ...                 by using Dashboard CRDs: OdhQuickStart, OdhDocument (for both how-to and tutorial)
+    ...                 and OdhApplication.
+    [Tags]    Tier2
+    ...       ODS-697    ODS-1768    ODS-1769    ODS-1770
+    Create Custom QuickStart
+    Create Custom Application
+    Create Custom How-To
+    Create Custom Tutorial
+    Check Custom QuickStart Item Has Been Successfully Created
+    Check Custom How-To Item Has Been Successfully Created
+    Check Custom Tutorial Item Has Been Successfully Created
+    Check Custom Application Item Has Been Successfully Created
```

Comment:
I believe` Check Custom Application Item Has Been Successfully Created` is failing because of a product bug. Could you add the tag ProductBug (and ProductBug ID in the [Documentation] section)?

Would it make sense to have 4 tests instead of 1 tests with 4 polarionids?

Reply:
the bug should be fixed now in the latest v1.18 build.

> Would it make sense to have 4 tests instead of 1 tests with 4 polarionids?

originally it was in that way. But all the resources depend on the custom application so instead of creting the custom application 4 times, I preferred to have the 4 tests in one